### ISSUE TITLE
Import of `orientation` fails since release of Numpy v2.0

### ISSUE THREAD

**Issue by fzag** [pyfar/pyfar#630]
## General

* pyfar version: 0.6.6
* Python version: 3.12.2
* Operating System: MacOS Sonoma 14.5 + Ubuntu 22.04.4
* Did you install pyfar via pip: yes

## Description

Import of `orientations` module fails with Numpy v2.0. (released on 16.06.2024; see [Numpy v2.0.0 release](https://github.com/numpy/numpy/releases/tag/v2.0.0))

Line where exception is raised:
https://github.com/pyfar/pyfar/blob/e566619b976e589cf4a5b091d0902725fea9bb86/pyfar/classes/orientations.py#L10

log:
```
Traceback (most recent call last):
  File "<stdin>", line 1 in <module>
  File [some/path]/numpy/__init__.py": line 410, in __getattr__
    raise AttributeError("module {!r} has no attribute "
AttributeError: module 'numpy' has no attribute 'VisibleDeprecationWarning'
```

## What I Did

This can be fixed by importing `VisibleDeprecationWarning` from `numpy.exceptions` instead of importing `numpy`.

Alternatively, numpy version could be pinned in the requirement file to `<2.0` until fixes are made.

## Notes:
This suggests there could be other parts of the repo which are not compliant with Numpy v2.0.

**Comment by f-brinkmann** [comment]
Thanks for reporting. This is because of numpy 2.0. We are fixing this soon in #629. Until then you can use `pip install numpy'<2.0'` in your environment.

**Comment by ahms5** [comment]
it is fixed in pyfar 0.6.7, which was just released
